lib/src/lib/table/components/table/table.component.ts

```diff
@@ -285,6 +286,8 @@ export class Lab900TableComponent<T extends object = object, TabId = string> imp
     }
   }
 
+  public hideSelectableCheckbox = (row: T): boolean => !!this.selectableRowsOptions?.hideSelectableRow?.(row);
+
```

Comment:
would it not be more efficient to calculate this once, and set the result in the component. 
That way you don't have to re calculate this function every render-frame. And it's not like the table data will change right?

Reply:
If the table data does change it should still only be run when the data is updated instead of every render. This could become very slow if you can input any function.

Reply:
@lab900-frederico-teixeira maybe just implement redux in our library? Thanks 👍